### PR TITLE
Fix #6190 - Change Log access from Document Detail View

### LINKED ISSUE #6190: Unable to access Change Log from Detail view when enabling auditing on Documents module
#### Issue
When enabling auditing on the Documents module, the 'View Change Log' button is not created in the Actions menu accessible from the Detail view. It does however enable the View Change Log button in the edit view. Documents_audit table is correctly created and auditing for the selected fields works as designed.

#### Expected Behavior
Enabling auditing for the Documents module should add the 'View Change Log' button to the Detail view for the module. As is the behavior when enabling auditing on Calls or Tasks.

#### Actual Behavior
No 'View Change Log' button accessible from Detail view after enabling auditing

```
<?php
$dictionary['Document']['audited'] = true;
?>
```
> In Studio, enable auditing for desired field (e.g. Status) 
> Run Quick Repair & Rebuild to enable auditing.
> Enter Document, Save, Edit audited field
> Open Documents Detail View > Actions Menu > no 'View Change Log' button present
> Open Documents Edit View > View Change Log > Confirm audit link created
> Check database for Documents_audit table, confirm created and logging changes for audited fields

#### Context
Require access to audit logs for an internal workflow, and tracking the approval status of a Document at current workplace. Must be able to check/verify appropriate personnel have approved the Document, without requiring users to enter Edit view and risk further changes

#### Your Environment
Tested with:
SuiteCRM 7.10.2 / Ubuntu 16.04
SuiteCRM 7.10.7 / Ubuntu 18.04
SuiteCRM 7.10.7 / Windows 10

### PR DESCRIPTION
**Pull request by ghost**
## Description
Small changes to the Document module's detailviewdefs file, and addition of detailHeader.tpl to enable the 'View Change Log' button in Documents Detail View when auditing is enabled for the module.

Previously, although auditing could be enabled the only way to access the change log / auditing table for the Documents module would be to enter Edit View or link manually with the id, per Issue #6190 

## Motivation and Context
Required internally to trace approvals process and changes to quotes & sales documents in conjunction with Workflow. 

Although the change log can be viewed by entering the Edit view, this is undesirable for a restricted document, and would require role changes to allow audit access for unassigned users.

```
<?php
$dictionary['Document']['audited'] = true;
?>
```
>- In Studio, enable auditing for desired field (e.g. Status) 
>- Run Quick Repair & Rebuild to enable auditing.
>- Enter new Document, Save, Edit audited field
>- Open Documents Detail View > Actions Menu > View Change Log

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist
- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: modules/Documents/tpls/detailHeader.tpl [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{*
+/*********************************************************************************
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by Salesagility Ltd.
+ * Copyright (C) 2011 - 2014 Salesagility Ltd.
```
Comment (samus-aran) [inline modules/Documents/tpls/detailHeader.tpl]: @jsamelko Can you update with appropriate header :+1: 
https://docs.suitecrm.com/community/contributing-code/coding-standards/

**Comment by samus-aran** [comment]
Would be good to have acceptance tests on this then apply acceptance tests for all modules that are audited = true.

File: modules/Documents/tpls/detailHeader.tpl (hunk 1 above)
Comment (jsamelko) [reply modules/Documents/tpls/detailHeader.tpl]: @samus-aran No problems - updated. Thanks

**Comment by pribeiro42** [comment]
Assessed

**Pull request merged**